src/commands/functions-secrets-set.ts

```diff
@@ -81,6 +82,12 @@ export const command = new Command("functions:secrets:set <KEY>")
         endpointsToUpdate.map((e) => `${e.id}(${e.region})`).join("\n\t"),
     );
 
+    if (options.nonInteractive && !options.force) {
+      throw new FirebaseError(
+        `Unable to destroy the stale version of secret ${secret.name} with the --force flag`,
```

Comment:
THe error message here seems to be for cases were --force is included, but this if statement covers cases where it is not included.